### PR TITLE
[ARTS-826] Standardise configuration files

### PR DESCRIPTION
**Pull request by tamirkamara**
## Description
Please replace this text with a summary of the change and which issue is fixed.

### Changes
- [ARTS-826] make all service config files more consistent with remarks
- Remove Netflix Ribbon from the spring cloud services (still present in the business services)
- Remove log4j leftovers

### Checklist:

- [x] Branch name follows convention (feature/arts-#-short-name OR fix/arts-#-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

[ARTS-826]: https://ndph-arts.atlassian.net/browse/ARTS-826

### CODE REVIEW

File: .ci/docker/check-docker-compose-services.sh [hunk 1]
```diff
@@ -16,14 +16,15 @@ is_healthy() {
         echo "$1 is healthy"
         return 0
     else
-        echo "Waiting for $1"
+        echo "Waiting for $1, logs tail is:"
+        docker-compose logs --tail=10 ${service}
```
Comment (eladiw) [inline .ci/docker/check-docker-compose-services.sh:20]: 👍

File: .ci/docker/check-docker-compose-services.sh [hunk 2]
```diff
@@ -16,14 +16,15 @@ is_healthy() {
...
         return 1
     fi
 }
 
 # on CI we want to test ALL services (discovery, config and gateway) using dev profile (instead of the local default)
 docker-compose pull -q
-docker-compose up --no-build -d gateway-server config-server discovery-server
+docker-compose up --no-build -d discovery-server config-server gateway-server
```
Comment (eladiw) [inline .ci/docker/check-docker-compose-services.sh:27]: whats the difference

Comment (tamirkamara) [reply .ci/docker/check-docker-compose-services.sh:27]: I thought to order it by importance that might cause the discovery to come online first (there might be priority/resourcing consideration)

File: docker-compose.yml [hunk 3]
```diff
@@ -177,7 +177,7 @@ services:
     ports:
       - "8761:8761"
     healthcheck:
-      test: curl http://discovery-server:8761/
+      test: curl http://localhost:8761/
```
[the remark this answers is not in the document]
Comment (tamirkamara) [reply docker-compose.yml:180]: yes, it works for all other services so I just aligned the two that weren't using localhost.

File: gateway-server/src/main/resources/application-local.yml [hunk 4]
```diff
@@ -1,3 +1,4 @@
+# why does gateway need to work in local mode?
```
[the remark this answers is not in the document]
Comment (tamirkamara) [reply gateway-server/src/main/resources/application-local.yml]: reverted for now

**Review by shajidissa — APPROVED** [review]

**Review by eladiw — APPROVED** [review]

**Pull request merged**